pubsub/rabbitmq/rabbitmq.go

```diff
@@ -401,6 +406,57 @@ func (r *rabbitMQ) prepareSubscription(channel rabbitMQChannelBroker, req pubsub
 		args[argMaxPriority] = mp
 	}
 
+	// queue type is classic by default, but we allow user to create quorum queues if desired
+	if val, ok := req.Metadata[metadataQueueType]; ok && val != "" {
```

Comment:
If queue type isn't specified at the request level, do you want to use the global component queue type you made configurable? `r.rabbitMQMetadata.QueueType`

Reply:
I will change it to indicate queue type on req level instead of component